share/spice/yacht_specs/yacht_specs.css

```diff
@@ -0,0 +1,21 @@
+/*
+    Document   : zeroclickinfo-spice/share/spice/yacht_specs/yacht_specs.css
+    Created on : 2015-07-09 10:31 AM
+    Author     : elephantRunAway
+    Description:Modification of the display of results for yacht related queries.
+*/
+
+    .thumb div {
```

Comment:
Please fix the indentation here. We use soft-tabs, 4 spaces. As well, please namespace this css by prefixing `.zci--yacht_specs` to each rule. We automatically add this class (based on the package name) to each IA's container so that the CSS will not affect the rest of the elements on that page (which in it's current state it would!)

```css
.zci--yacht_specs .thumb div {
    ...
    ...
}
...
```